specificatie/brp-api/gezag/derde-v1.yaml

```
@@ -0,0 +1,33 @@
+openapi: 3.1.0
+info:
+  title: derde type definitie
+  version: 2.0.0
+paths: {}
+components:
+  schemas:
+    AbstractDerde:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          Derde: '#/components/schemas/Derde'
+          OnbekendeDerde: '#/components/schemas/OnbekendeDerde'
+            
+    Derde:
+      allOf:
+        - $ref: '#/components/schemas/AbstractDerde'
+        - type: object
+          properties:
+            burgerservicenummer:
+              $ref: '../../brp/identificatienummers/burgerservicenummer-v1.yaml#/components/schemas/Burgerservicenummer'
+            naam:
+              $ref: '../naam/volledige-naam-v1.yaml#/components/schemas/NaamVolledigeNaam'
+
+    OnbekendeDerde:
+      allOf:
+        - $ref: '#/components/schemas/AbstractDerde'
```

Ik zou graag de types willen hernoemen naar
- derde
- bekende derde
- onbekende derde

Dat heeft naar mijn mening een natuurlijkere transitie naar gevoerde taal (tijdens dag start, overleg, etc.). 

```suggestion
    Derde:
      type: object
      required:
        - type
      properties:
        type:
          type: string
      discriminator:
        propertyName: type
        mapping:
          Derde: '#/components/schemas/BekendeDerde'
          OnbekendeDerde: '#/components/schemas/OnbekendeDerde'
            
    BekendeDerde:
      allOf:
        - $ref: '#/components/schemas/Derde'
        - type: object
          properties:
            burgerservicenummer:
              $ref: '../../brp/identificatienummers/burgerservicenummer-v1.yaml#/components/schemas/Burgerservicenummer'
            naam:
              $ref: '../naam/volledige-naam-v1.yaml#/components/schemas/NaamVolledigeNaam'

    OnbekendeDerde:
      allOf:
        - $ref: '#/components/schemas/Derde'
```

---

Resolved in a2acaf576e3cee7254f3b3f4fb36716259449201